src/style.css

```diff
@@ -0,0 +1,142 @@
+/* stylelint-disable shorthand-property-no-redundant-values */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: content-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+.input {
+  position: relative;
+  height: 100%;
+  width: calc(100% - 60px);
+  font-size: 16px;
+  font-weight: 300;
+  left: 16px;
+  top: -1px;
+  outline: none;
+  border: none;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+.image {
+  position: relative;
+  top: 3px;
+  left: 5px;
+}
+
+.search-bar {
+  position: relative;
+  margin: 20px 8px;
+  padding: 20px;
+  font-size: 24px;
+  height: 28px;
+  max-height: 70px;
+  line-height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.input::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input:focus {
+  font-family: "AvenirHeavy", sans-serif;
+  font-weight: 800;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  outline: none;
+  border: none;
+}
+
+.search-bar--small {
+  position: relative;
+  margin: 0 8px;
+  padding: 0;
+  height: 40px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  font-family: "Avenir", sans-serif;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+  max-height: 70px;
+}
+
+.input--small {
+  position: inherit;
+  top: 1px;
+  left: 18px;
+  height: 36px;
+  line-height: 19px;
+  font-weight: 400;
+  font-size: 14px;
+  font-family: "Avenir", sans-serif;
+  width: calc(100% - 36px);
+  outline: none;
+  border: none;
+}
```

Comment:
Same here. Do not repeat your self. 

![2023-04-27_15h51_41](https://user-images.githubusercontent.com/61904995/234867367-4156ae9d-b464-4cbd-b307-91c6d6aef0bd.png)